packages/react-devtools-shared/src/backend/types.js

```diff
@@ -523,3 +524,10 @@ export type ConsolePatchSettings = {
   hideConsoleLogsInStrictMode: boolean,
   browserTheme: BrowserTheme,
 };
+
+export type DevToolsHookSettings = {
+  appendComponentStack: boolean,
+  breakOnConsoleErrors: boolean,
+  showInlineWarningsAndErrors: boolean,
+  hideConsoleLogsInStrictMode: boolean,
+};
```

Comment:
This will be a successor of `ConsolePatchSettings`, the only difference is `browserTheme` field, but looks like it is not used. I will do it later in this stack.